crates/compiler/builtins/roc/Num.roc

```diff
@@ -485,17 +485,32 @@ F32 : Num (FloatingPoint Binary32)
 ## convert any [U64] to a [Dec] without losing information.
 ##
 ## There are some use cases where [F64] and [F32] can be better choices than [Dec]
-## despite their precision issues. For example, in graphical applications they
+## despite their lower precision. For example, in graphical applications they
```

Comment:
we still may want better wording here. Depending on the exact range of your numbers, floats could have higher precision than dec. Dec has higher precision for numbers with an absolute value greater than 0.01 roughly. Anything smaller should have higher precision with doubles. And anything with values larger than max dec of course can be represented in float, but not dec. Otherwise, they map correctly to base 10 and that isn't exactly percion. Just wonder if there is a better way to capture this nuance.

Reply:
Good point! I took a pass at revising it in 83587eca222dbe4fd780687114bff4f1f364ff59